src/Raven.Client/Http/RequestExecutor.cs

```diff
@@ -442,6 +442,12 @@ private void UpdateTopologyCallback(object _)
                     _lastKnownUrls = initialUrls;
                     throw;
                 }
+                catch (DatabaseDoesNotExistException e)
+                {
+                    if (initialUrls.Length == 1)
```

Comment:
This is wrong, this should always throw. 
`DatabaseDoesNotExistsException` is not recoverable.

Reply:
What happens if you have more the one url in the store, and in one of those url the database exists?

Reply:
What happens if you have more then one url in the store, and in one of those url the database exists?